rdm/init_files/documents/software_plan.md

```diff
@@ -48,9 +48,10 @@ The activites described in this document are designed for a team composed of a p
 
 At least one team member must be trained in risk management [[14971:3.3]].
 
+
 ## Related Documents
 
-[[:This section fulfills 62304:5.1.8]]
+[[:This section fulfills 62304:5.1.8.a 62304:5.1.8.b 62304:5.1.8.c]]
```

Comment:
```suggestion
[[:This section fulfills 62304:5.1.8.a, 62304:5.1.8.b, and 62304:5.1.8.c]]
```

Reply:
I think all of these lists of references should be in sentence form. What do you think?